src/includes/getting-started-primer/unreal.mdx

```diff
@@ -11,6 +11,12 @@ This SDK is a preview that packages Sentry's native SDKs into an Unreal Engine p
 - Native support for automatic crash error tracking for
   - Android by using the [Android SDK](/platforms/android/) to support Java, Kotlin, C and C++
   - iOS by using the [iOS SDK](/platforms/apple/guides/ios/) to support Objective-C, Swift, C and C++
-  - Windows and Mac by using the [Native SDK](/platforms/native/) to support C and C++ with minidumps.
+  - Windows, Mac and Linux by using the [Native SDK](/platforms/native/) to support C and C++ with minidumps
 - Compatible with Unreal Engine 5
 - [Crash Report Client](/platforms/unreal/setup-crashreport/) provided along with Unreal Engine
+
+<Alert level="warning" title="Warning">
+
+In order to capture crashes on Windows and Mac automatically Crash Report Client has to be configured.
```

Comment:
```suggestion
In order to capture crashes on Windows and Mac automatically [Crash Reporter Client](setup-crashreporter/) has to be configured.
```

We need to rename the page to Setup Crash Reporter Client. And add a redirect from the old name